docs/DeveloperGuide.md

```diff
@@ -921,36 +921,242 @@ testers are expected to do more *exploratory* testing.
 
    1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-1. Saving window preferences
+2. Saving window preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+3. Exiting the program
+
+    1. Enter `exit` command in command input box or click on X button.<br>
+       Expected: The application closes
+
+### Adding a stall
+
+1. Adding a stall
+
+    1. Prerequisites: None
+
+    2. Test case: `sadd n/Ah Kim Chicken Rice a/311, Clementi Ave 2, #02-25 t/chickenRice t/opensDaily`<br>
+       Expected: A new stall is added to the list. Details of the newly added stall shown in the status message. GUI updates to show the newly added stall and its details.
```

Comment:
Note: the added tags are stored as chickenrice and opensdaily. Might want to clarify that because it's not obvious here.

Reply:
Good idea to include that the tags are saved in lowercase just in case

Reply:
Fixed.